app/MindWork AI Studio/Components/ChatComponent.razor

```diff
@@ -80,6 +80,11 @@
                     <MudIconButton Icon="@Icons.Material.Filled.CommentBank" OnClick="() => this.StartNewChat(useSameWorkspace: true)"/>
                 </MudTooltip>
             }
+            
+            @if (this.SettingsManager.ConfigurationData.ChatTemplates.Count > 0)
+            {
+                <ChatTemplateSelection CurrentChatTemplate="@this.currentChatTemplate" CurrentChatTemplateChanged="@this.ChatTemplateWasChanged"/>
```

Comment:
Okay, if I understand correctly, this is how it works: We display a new icon for the chat templates. I thought we would dynamically replace the three existing buttons, like we discussed. I need to take a look at this solution in the compiled app to form an opinion.